src/components/Menu/Menu.tsx

```diff
@@ -75,8 +75,9 @@ export const Menu = ({
           className={menuClassNames}
           defaultOpenKeys={defaultOpenKeys}
           defaultSelectedKeys={defaultSelectedKey ? [defaultSelectedKey] : undefined}
+          getPopupContainer={() => document.querySelector(`.${menu}`)!}
           inlineCollapsed={isCollapsed}
-          inlineIndent={12}
+          inlineIndent={0}
```

Comment:
If the value is 0 we can also put it straight as a constant. 
I wouldn't do the same with other values since the value might change depending on the theme configuration.

Reply:
For our use cases we have to date given up the use of the padding provided by Antd.